mds/serializers.py

```diff
@@ -186,25 +186,40 @@ def update(self, instance, validated_data):
         return instance
 
 
-class ServiceAreaSerializer(BaseModelSerializer):
+class PolygonSerializer(BaseModelSerializer):
     """A service area
     """
 
     id = serializers.UUIDField(
-        required=False, help_text="Unique Area identifier (UUID)"
+        required=False, help_text="Unique Polygon identifier (UUID)"
     )
-    # TODO(lip) remove default
-    provider = serializers.UUIDField(
-        default="a19cdb1e-1342-413b-8e89-db802b2f83f6",
-        help_text="A unique ID identifying the service provider (UUID)",
+    label = serializers.StringRelatedField(help_text="Name of the polygon")
+    creation_date = serializers.DateTimeField(help_text="Polygon creation date")
+    deletion_date = serializers.DateTimeField(
```

Comment:
do we also need creation/deletion dates on polygons themselves ? If yes, maybe explain the difference with the dates on the Area model in a comment ?